ArduPlane/tiltrotor.cpp

```diff
@@ -229,7 +231,9 @@ void Tiltrotor::continuous_update(void)
 
         max_change = tilt_max_change(false);
 
-        float new_throttle = constrain_float(SRV_Channels::get_output_scaled(SRV_Channel::k_throttle)*0.01, 0, 1);
+        SRV_Channels::set_output_scaled(SRV_Channel::k_throttle_tilt, SRV_Channels::get_output_scaled(SRV_Channel::k_throttle));
```

Comment:
I think this should constrain to not be negative before the set. Normal throttle can be negative. 
```suggestion
        SRV_Channels::set_output_scaled(SRV_Channel::k_throttle_tilt, MAX(0, SRV_Channels::get_output_scaled(SRV_Channel::k_throttle)));
```

Reply:
done